numpy/core/getlimits.py

```diff
@@ -359,6 +360,13 @@ class finfo:
     impacts import times.  These objects are cached, so calling ``finfo()``
     repeatedly inside your functions is not a problem.
 
+    Note that `tiny` is not actually the smallest positive representable
+    number in a NumPy floating point type. As in the IEEE-754 standard,
+    NumPy floating point types make use of subnormal numbers to fill the
+    gap between 0 and `tiny`. However, subnormal numbers may have a 
+    significantly reduced precision. Compare further with the Wikipedia page
+    `Denormal Numbers <https://en.wikipedia.org/wiki/Denormal_number>`_
+    
```

Comment:
```suggestion
    significantly reduced precision[1]_.
    
    References
    ----------
    .. [1] Wikipedia, "Denormal Numbers",
           https://en.wikipedia.org/wiki/Denormal_number
```

Reply:
Again, this is just my opinion, but I think an explicit reference reads better than a link with some connecting text (you may have to check my indentation - it's difficult to get right in the suggestion feature)